src/library_browser.js

```diff
@@ -783,17 +779,18 @@ var LibraryBrowser = {
     }
   },
 
-  emscripten_async_wget__deps: ['$PATH_FS'],
+  emscripten_async_wget__deps: ['$PATH_FS', '$runtimeKeepalivePush', '$runtimeKeepalivePop'],
   emscripten_async_wget__proxy: 'sync',
   emscripten_async_wget__sig: 'viiii',
   emscripten_async_wget: function(url, file, onload, onerror) {
-    noExitRuntime = true;
+    runtimeKeepalivePush();
 
     var _url = UTF8ToString(url);
     var _file = UTF8ToString(file);
     _file = PATH_FS.resolve(_file);
     function doCallback(callback) {
       if (callback) {
+        runtimeKeepalivePop();
         var stack = stackSave();
         {{{ makeDynCall('vi', 'callback') }}}(allocate(intArrayFromString(_file), ALLOC_STACK));
```

Comment:
this is a user callback, so doesn't it need the wrapper?

Reply:
I'd was planning on increasing the use of `callUserCallback` in various followup PRs.  I see this PR as an initial step in the direction, to test the water.